### ISSUE TITLE
performance 42 worlds

### ISSUE THREAD

**Issue by isoz** [RedCraftGH/RedSkyBlock#31]
![image](https://user-images.githubusercontent.com/6126350/60789339-c96af480-a156-11e9-9ed6-b1d77126471a.png)

Having some lag issues here.

**Comment by RedCraftGH** [comment]
Heyo! 42 worlds are quite a few worlds! I can definitely understand why there may be some performance problems there. I will, of course, be trying my best to make the performance of this plugin better in the future, but as of now, there isn't a ton that I can do to fix this for you. I guess the only thing I can really suggest for you now is to increase the amount of power you have allotted to your server.

**Comment by isoz** [comment]
Nah. I had over a thousand worlds with Giantquartz skyblock performance was
ok, its just bad code with an even worse database. It takes too much time
to access a file this big and the playermoveevent also being one of the
main culprits overall.

**Comment by RedCraftGH** [comment]
If you wish to use the plugin then the performance issues are something you will have to deal with for now, otherwise you can use another plugin. As I said, improving the performance of the plugin is something I will be working on in the future. Thanks for reporting your problem.

**Comment by isoz** [comment]
That's alright, I know the consequences, I'm just reporting my findings to you and to whoever hangs around here. 

I'm already working on improving this but getting rid of the playermoveevent is the most important step to reduce  load in the main thread.
